src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/DirectionDetailsCollection.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class DirectionDetailsCollection {
```

Comment:
new models to map CCD collection to, we should be doing this for all of our collections to improve our typing rather than using maps everywhere